docs/interactions/Application_Commands.md

```diff
@@ -1077,24 +1077,21 @@ Fetch a global command for your application. Returns an [application command](#D
 
 ## Edit Global Application Command % PATCH /applications/{application.id#DOCS_RESOURCES_APPLICATION/application-object}/commands/{command.id#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object}
 
-> info
-> All parameters for this endpoint are optional.
```

Comment:
I don't see how these indicators are 'redundant' if you remove both things mentioning they are optional?

Reply:
Err, I didn't. Read the description for the request.

Reply:
no, it's redundant because it says "All fields are optional" right below this. fields are not nullable by default